src/components/organisms/sidebar/sidebar.tsx

```diff
@@ -101,36 +100,35 @@ export const Sidebar = () => {
 					</div>
 
 					<div className="flex flex-col justify-end gap-5">
-						{featureFlags.systemLog ? (
-							<Button
-								className="hover:bg-green-200"
-								onClick={() => toggleLogger(true)}
-								title={t("systemLog")}
+						<Button
+							className="hover:bg-green-200"
+							onClick={() => toggleLogger(true)}
+							title={t("systemLog")}
```

Comment:
Please add the same systemLog title to the buttons ariaLabel

Reply:
added